app/code/AKisilenko/ModuleLesson6/view/frontend/web/js/ask-question.js

```diff
@@ -0,0 +1,82 @@
+define([
+    'jquery',
+    'Magento_Ui/js/modal/alert',
+    'mage/cookies',
+    'jquery/ui'
+], function ($, alert) {
+    'use strict';
+
+    $.widget('moduleLesson6.AskQuestion', {
+        options: {
+            cookieName:'time_cookie'
+        },
+
+        /** @implements*/
+        _create: function () {
+            $(this.element).submit(this.submitForm.bind(this));
+        },
+
+        /**
+         * Submit form
+         */
+        submitForm: function () {
+
+            if (!this.validateForm()) {
```

Comment:
Cookie should be checked before sending form. DO not send data if the cookie is still present in 120 seconds.